index.html

```diff
@@ -121,36 +121,103 @@
       </h2>
       <p>
         The API provides bunch of interfaces that are used to express the
-        semantics of web content in a that assistive technologies (AT) knows
-        how to deal with. In other words, each piece of semantically meaningful
-        content has associated accessible object the AT operates on.
+        semantics of web content in a way that assistive technologies (AT)
+        knows how to deal with. Also it provides ability to search and traverse
+        the content and interact with it. It has a way to extend existing
+        semantics of the markup, and add new semantics for inaccessible content
+        like drawings made using the <code>canvas</code> element. Each piece of
+        semantically meaningful content has associated accessible object the AT
```

Comment:
s/semantically meaningful/semantic

"semantically meaningful" is tautological.